### PR TITLE
Astro add docs

### PR DESCRIPTION
**Pull request by aFuzzyBear**
The premise of this PR is to introduce the a new cli function `astro docs`. 

The purpose of this utility function is to launch the systems browser to point to the Astro Docs site.

## Progress

`browse(url)` - this is the main function that launches the browser.

So far, it is agnostic to the following platforms;
- Android,
- Linux,
- MacOS
- Windows
- Gitpod
- Stackblitz

It doesnt work on CodeSandbox, as I was unable to find their own built in browser function. The only other thought I had to overcome this is to sniff out the CSB platform and have it open a new window and point it that way.  

That is the method works, just cant seem to tie it into the cli properly, I would need to spend a bit more time on that tomorrow. 

If all else fails Ill try to apply the 'open' package by [sindresorhus](https://github.com/sindresorhus/open), If I continue to have no luck,

## Changes

- Introduces a new CLI command `astro docs`
- One short launch to the Docs 

//TODO:  Don't forget a changeset! `pnpm exec changeset`

## Testing
I really need to figure this out, perhaps once I get the actual cli bit to work, 

## Docs
Yet to be written but it should be simple.

---
```md
## astro docs

A single command to launch your systems browser pointing to Astro's Doc site. This is to help provide a fast way to launch the Doc site directly from your terminal! 
```

Something along that line, since there isnt much to it. 

---

## Considerations

I am considering the idea of a `astro browser <url>` CLI utility, I find that this could just be a novelty feature that may serve no other purpose than to give those terminal dwellers a fun way of launching a website from within their Astro project,

### CODE REVIEW

**Review by natemoo-re — COMMENTED** [review]
This is cool! Did you borrow this from somewhere? Might be nice to add a comment to link back to the source.

**Comment by aFuzzyBear** [comment]
I will do, mainly it was the commands that I found but I will link it back to the source code where I got it from =)
Just going to see about tying this into the cli tool, 
Would it be out of whack to make an `astro browse <url>` ? it wouldnt be difficult at all to implement

**Comment by aFuzzyBear** [comment]
Is it possible for someone to help me with getting this simple method over the line, 
I have added it as best as I could to the CLI tool, as where I believe it should go, however the issue is that the `browser()` is now not launching properly, at least when I have tried to test this on Gitpod, 

I do have some questions, 
The first is what am I doing wrong, if there is anything, please do feed this back to me, 
How/Where would I learn how to write tests for this.
If there is anything else please do let me know,

**Comment by aFuzzyBear** [comment]
It now works !!!

I had miss read the cli function properly, when I realised it was a very simple fix, and it now works a peach!!

 🚀👨‍🚀📚 `astro docs` ladies and fellow astronauts is by all accounts good to go, 🚀👨‍🚀📚

File: packages/astro/src/cli/index.ts [hunk 1]
```diff
@@ -60,6 +60,7 @@ async function printVersion() {
 function resolveCommand(flags: Arguments): CLICommand {
 	const cmd = flags._[2] as string;
 	if (cmd === 'add') return 'add';
+	if (cmd === 'docs') return 'docs';
 
 	if (flags.version) return 'version';
 	else if (flags.help) return 'help';
```
Comment (JuanM04) [inline packages/astro/src/cli/index.ts:66]: I'd remove that `if` and add `'docs'` to the `supportedCommands` set. That what, if anyone does `astro docs --help`, they'll see the full help message!

**Review by JuanM04 — CHANGES_REQUESTED** [review]
Awesome, I love it! I've left some comments

**Review by aFuzzyBear — COMMENTED** [review]
Applied @JuanM04 's recommendations and have moved the `cmd('docs')` to the `supportedCommands()` Also resolved the import conflicts from the main branch,

**Comment by aFuzzyBear** [comment]
Kinda scared of the `changeset` it was asking if I wished to make a `major bump` of `0.25.4` and I didnt wish to regress the hard work of others, so I really have left this alone, sorry,

**Pull request merged**